core/eolearn/core/eodata_io.py

```diff
@@ -285,6 +287,25 @@ def walk_filesystem(filesystem: FS, patch_location: str, features: FeaturesSpeci
     return result
 
 
+def walk_filesystem(
```

Comment:
i'd decorate with a deprecation decorator, since it will be removed at some point

Reply:
don't think we have decorators for this in eo-learn, so I just added the deprecation warning manually.